packages/components/src/components/entities/SingleParentEntityPanel.tsx

```diff
@@ -192,7 +191,7 @@ export class SingleParentEntityPanel extends React.Component<Props, State> {
                     <QuerySelect
                         componentId={'parentEntityValue_' + lcTypeName} // important that this key off of the schemaQuery or it won't update when the SelectInput changes
                         containerClass="row"
-                        disabled={lcTypeName === undefined}
+                        disabled={false}
```

Comment:
I assume false is the default value for `disabled` so could this just be removed?